drafts/draft.md

```diff
@@ -156,6 +156,33 @@ Topics: Technologies enabling democratic participation, e.g. online voting, liqu
 -  Gender equity and inclusion in research and innovation
 
 
+organizational overview:
+
+- we need democratic control over the information / input we provide, which is used for technological ends
+  - data autonomy
+  - biometric data, different kinds of data (content we put online, ...)
+  - ? support (require?) open protocols for data ownership / data commons
+  - ? ban sale of third-party data
+  - ? carlo's proposals: ban personal data collection by proprietary agents / mandate only device-local algorithms for personalization
+- we need democratic control over the ways in which we are subjected to technology
+  - algorithmic accountability + bias (+ some definition of scope)
+  - targeting / exclusion (two-sided harms relating to representation or lack thereof in data)
+  - worker surveillance
+  - accessibility
+  - 
+- we need democratic control over the research and knowledge that powers technology
```

Comment:
Also to include in this section:

- topics on education (e.g. proposal #5)
- crowdfunding innovation platform (#14)